module/userProjects/helper.js

```diff
@@ -1113,7 +1114,28 @@ module.exports = class UserProjectsHelper {
                         solutionDetails = solutionDetails.data[0];
                         
                     }
-
+                    // program join API call 
```

Comment:
@ankitpws Don't we need a appversion check here? I think for app version less than 5.2 only we require to call program join internally. from 5.2 onwards app team will be calling the new join api right?